test/metabase/models/card_test.clj

```diff
@@ -596,6 +596,25 @@
       (is (= #{["Card" (:id card1)]}
              (serdes/descendants "Card" (:id card2)))))))
 
+(deftest extract-test
```

Comment:
I understand that there are serdes v2 extraction tests in other test namespaces, but I think this is the appropriate place for unit testing the per-model extraction functions.